### ISSUE TITLE
Call Details Dropped in Labs Site

### ISSUE THREAD

**Issue by zang0** [socrata/labs.socrata.com#2]
Comparing: 

http://labs.socrata.com/docs/search.html#-searching-particular-domains-get

to screen shot below. Note: helpful to actually see a usable URL w/ the domain as well as example call parameters entered. Also helpful to see details on the parameters.

![screen shot 2015-04-22 at 10 44 18 am](https://cloud.githubusercontent.com/assets/1051776/7281160/9a7bcb72-e8dc-11e4-925e-f9f455b446b1.png)

**Comment by chrismetcalf** [comment]
Yeah, the [aglio](https://github.com/danielgtaylor/aglio) templates are very different from the Apiary docs, so we'll have to figure out how to get those in there somehow.
